shaders/billboard.glsl

```diff
@@ -0,0 +1,47 @@
+#version 150
+// Copyright 2015, Christopher J. Foster and the other displaz contributors.
+// Use of this code is governed by the BSD-style license found in LICENSE.txt
+
+// Draws billboards, which are textured quads that always face the camera,
```

Comment:
I don't think these are billboards in the usual usage of the term.  I suggest renaming the shader `annotations.glsl`